### PR TITLE
Showing Usage Of Session Override (Codeception 2.5)

### PR DESCRIPTION
**Pull request by Fenikkusu**
Hello!

* Type: new feature | documentation

This pull request affects the following components: **(please check boxes)**

* [x] Project Core
* [ ] Code Style
* [x] Documentation
* [x] Testing

**In raising this pull request, I confirm the following (please check boxes):**

- [ ] I have read and understood the [Contributing Guidelines](https://github.com/Codeception/phalcon-demo/blob/master/CONTRIBUTING.md)?
- [ ] I have checked that another pull request for this purpose does not exist.
- [ ] I wrote some tests for this PR.

Small description of change:

Adding testing to demonstrate new feature for adding custom session object when running Codeception tests.

Thanks

### CODE REVIEW

File: tests/functional/SessionCest.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+class SessionCest
+{
+    public function testSession(FunctionalTester $I)
+    {
+        $I->assertTrue($I->grabServiceFromContainer('session')->getTrue());
```
Comment (sergeyklay) [inline tests/functional/SessionCest.php:7]:
```
PHP Fatal error: 
Call to undefined method Codeception\Lib\Connector\Phalcon\MemorySession::getTrue()
in /home/travis/build/Codeception/phalcon-demo/tests/functional/SessionCest.php on line 7
```

**Review by sergeyklay — CHANGES_REQUESTED** [review]

File: tests/functional/SessionCest.php (hunk 1 above)
Comment (DavertMik) [reply tests/functional/SessionCest.php:7]: Yes, this feature is about to come into Codeception 2.5.1 so test is not passing yet

Comment (DavertMik) [reply tests/functional/SessionCest.php:7]: See https://github.com/Codeception/Codeception/pull/5232

Comment (sergeyklay) [reply tests/functional/SessionCest.php:7]: @DavertMik As you can see the build has failed ¯\_(ツ)_/¯

**Comment by ruudboon** [comment]
Now that V2 is released Codeception runs on v4 I'm closing this.

**Pull request closed without merging**